index.html

```diff
@@ -24,15 +24,17 @@ <h1>Welcome to the student showcase!</h1>
         <div>FOUR</div>
         <div>FIVE</div>
         <div>
-        <img src="https://avatars0.githubusercontent.com/u/38234207?s=400&u=ff4f764c6655251d3f10e610b76d2141a584d989&v=4" alt="My photo">
+          <p>
```

Comment:
the `<p>` tag should be used to wrap text only. You should be using this on line `<30>` instead!